evaluation/tasks/tydiqa_secondary/tydiqa_secondary.py

```diff
@@ -0,0 +1,96 @@
+# Module for any additional processing required for the TyDi QA dataset
+# HuggingFace dataset link: https://huggingface.co/datasets/tydiqa
+from typing import Dict
+
+from jinja2 import Template
+from torch.utils.data import Dataset
+from datasets import load_dataset
+from tqdm import tqdm
+
+from evaluation.tasks.auto_task import AutoTask
+
+TEMPLATE = Template(
```

Comment:
When the multilingual prompts are ready, this chunk will be much longer. It'd be great to have a `load_template()` that returns a python dict with pairs of key `language` and value `jinja2.Template`.